### DISCUSSION TITLE
Not able to have Listing result

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#2831] [Q&A] [answered]
Hi,

I have this qmd sample file, to show the list of qmd files in params folder:

```
---
title: "Listing Example"
listing:
  contents:
    - "params/*.qmd"
  type: table
---

You can review the following documents for additional information:
```

But I have an empty page. I have the files in the folder. Do I must insert in them some specific meta info?

![image](https://user-images.githubusercontent.com/30607/195583187-102947c8-1df1-4af4-8e61-0274ebe85d0e.png)

I have quarto 1.2.210

Thank you

**Comment by dragonstyle** [comment, marked as answer]
Files that are completely devoid of metadata will be ignored in listings. We minimally need to be able to formulate a title (and ideally an author, description, and so on). Are the files in the `params` folder relatively normal `qmd` files?

**Reply by aborruso** [reply]
thank you

**Reply by aborruso** [reply]
I have opened a related PR https://github.com/quarto-dev/quarto-web/pull/443
